conda_smithy/feedstock_content/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Thank you for pull request.
+
+Below are a few things we ask you kindly to self-check before getting a review.
+
+* [ ] Bump the build number (if the version is unchanged)
```

Comment:
One more point would be `I used a fork in github`

Reply:
Might even be good in the contributing document too. Though this is certainly where I expect most people will see it first.